docs/api/scenario-basics.md

```diff
@@ -0,0 +1,356 @@
+  "title": "API",
+  "area_code": "nl",
+  "start_year": 2019,
+  "end_year": 2050,
+  ...
+  "user_values": {
+    "buildings_insulation_level": 40.3,
+    "capacity_of_energy_power_hydro_river": 39.0
+  }
+}
+```
+
+## Create a Scenario
+
+A scenario can be created in three ways: as a blank scenario for an area and end year, as a new scenario
+with certain sliders set, or as a scenario based on another scenario (its preset). All creation
+methods use a POST request on the scenario endpoint, but with different data.
+
+### Create a blank Scenario
+
+Creates a scenario where all sliders are at their default position. The following data can
+be supplied:
+
+* `area_code` - the code of the desired area, default: 'nl'
+* `title` - title of the scenario, default: 'API'
+* `end_year` - end year of the scenario, integer, default: 2015
+* `source` - identifier for the application creating the scenario (highly recommended), default: null
+
+<ApiEndpoint data={endpointData.create} />
+
+```http title="Example request"
+POST /api/v3/scenarios HTTP/2
+Host: engine.energytransitionmodel.com
+Accept: application/json
+
+{
+  "scenario": {
+    "area_code": "UKNI01_northern_ireland",
+    "end_year": "2040"
+  }
+}
+```
+
+```http title="Example response"
+HTTP/2 200 OK
+Content-Type: application/json; charset=utf-8
+
+{
+  "id": 123456,
+  "title": "API",
+  "area_code":"UKNI01_northern_ireland",
+  "start_year": 2018,
+  "end_year": 2040,
+  "url": "https://engine.energytransitionmodel.com/api/v3/scenarios/123456",
+  "ordering": null,
+  "display_group": null,
+  "scaling": null,
+  "source": null,
+  "template": null,
+  "created_at": "2021-07-16T09:23:00.000Z",
+  "updated_at": "2021-07-16T09:23:00.000Z",
+  "protected": false,
+  "esdl_exportable": null
+}
+```
+
+### Create a Scenario with slider settings
+
+User values may also be provided on create in the `user_values` object. You can supply the `detailed`
+parameter to see them reflected in the response.
+
+<ApiEndpoint data={endpointData.create} />
+
+```http title="Example request"
+POST /api/v3/scenarios?detailed=true HTTP/2
+Host: engine.energytransitionmodel.com
+Accept: application/json
+
+{
+  "scenario": {
+    "area_code": "UKNI01_northern_ireland",
+    "end_year": "2040"
+    "user_values": {
+      "buildings_insulation_level": 40.3,
+      "capacity_of_energy_power_hydro_river": 39.0
+    }
+  }
+}
+```
+
+```http title="Example response"
+HTTP/2 200 OK
+Content-Type: application/json; charset=utf-8
+
+{
+  "id": 123456,
+  "title": "API",
+  "area_code":"UKNI01_northern_ireland",
+  "start_year": 2018,
+  "end_year": 2040,
+  ...
+  "user_values": {
+    "buildings_insulation_level": 40.3,
+    "capacity_of_energy_power_hydro_river": 39.0
+  }
+}
+```
+
+:::warning Balancing groups
+Some sliders are grouped in a 'share group' to together sum to 100%. An error will be returned if
+you try to set all inputs in such a group without making sure the group still makes exactly 100.0.
+:::
```

Comment:
I have a bad feeling that this isn't true. I think share groups are only validated when *updating* a scenario. I've created an ETEngien issue: https://github.com/quintel/etengine/issues/1186

We can leave this text in if we can fix this in ETEngine soon; if not, I think it should be moved to the "Set sliders in a Scenario" section.